### DISCUSSION TITLE
How to filter on joined table (1 to 1) on multiple fields ?

### DISCUSSION THREAD

**Discussion by lakshit-shah-95** [prisma/prisma#17262] [Q&A] [answered]
model contact {
    id                    String                       @id
    first_name      String
    address_id     String
    property        property?                    @relation(fields: [address_id], references: [id], onDelete: NoAction)
 }
 
model property {
  id              String        @id
  city            String?
  state          String?
  country      String?
  postal        String?
  contact      contact[]
}

and this is the query i am trying
`return prisma.contact.findMany({
    select: {
      id: true,
      property: {
        select: {
          city: true,
          postal: true,
          country: true
        }
      }
    },
    where : {
      OR: [
        {
          property: {
            OR: [
              {
                city: {
                  equals: 'indore',
                  mode: 'insensitive'
                }
              }
            ]
          }
        },
        {
          property: {
            OR: [
              {
                postal: {
                  equals: 12345
                }
              }
            ]
          }
        },
        {
          property: {
            OR: [
              {
                country: {
                  equals: 'India',
                  mode: 'insensitive'
                }
              }
            ]
          }
        }
      ]
    }
  })`

and getting this error

PrismaClientValidationError: Unknown arg `OR` in where.OR.1.property.OR for type PropertyRelationFilter. Did you mean `is`?
at Document.validate (C:\Users\a\lambda_prisma\node_modules\@prisma\client\runtime\index.js:28329:20)

can anyone help with what's wrong here ?

**Comment by nurul3101** [comment, marked as answer]
Hello @lakshit-shah-95 👋 

I noticed that there is no `postal` column in the `property` model, but you seem to refer it in your query due to which you are getting this error. Can you try invoking `npx prisma generate` after adding the postal field and check if you get the same error?

**Reply by lakshit-shah-95** [reply]
Ahh, my bad on that. It actually is present in the model. I missed that while copying partial table structure. I have updated the question but the error is same. @nurul3101

**Reply by lakshit-shah-95** [reply]
hey @nurul3101 , this is fixed. It was issue to from our end in where condition.
this is what i was trying for string postal column
` { property: { OR: [ { postal: { equals: 12345 } `

this is the current working condition for postal column
` { property: { OR: [ { postal: { equals: "123456" } `

for string column, I was supposed to pass data in quotes, so my bad on that.

Thanks for replying @nurul3101

**Reply by nurul3101** [reply]
No worries! I am glad to hear that you were able to solve it! 🙌 
For reference, In case someone else comes across this, here is the [documentation](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#or) for OR operator.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
